README.md

```diff
@@ -84,7 +82,7 @@ tool_node = ToolNode(tools)
 model = ChatAnthropic(model="claude-3-5-sonnet-20240620", temperature=0).bind_tools(tools)
 
 # Define the function that determines whether to continue or not
-def should_continue(state: MessagesState) -> Literal["tools", END]:
+def should_continue(state: MessagesState) -> Union[Literal["tools"], type(END)]:
```

Comment:
let's instead replace END w/ `__end__` (also might as well update in all docs with sed!)